src/test/java/htsjdk/samtools/CRAMFileBAIIndexTest.java

```diff
@@ -51,43 +51,43 @@
     @Test
     public void testConstructors () throws IOException {
         CRAMFileReader reader = new CRAMFileReader(cramFile, indexFile, source, ValidationStringency.SILENT);
-        CloseableIterator<SAMRecord> iterator = reader.queryAlignmentStart("chrM", 1500);
+        CloseableIterator<SAMRecord> iterator = reader.queryAlignmentStart("chrM", 1519);
```

Comment:
Using a named constant instead of a magic number would avoid the need to modify it in many places the next time this changes. And also make it more apparent that the same value is used.

Reply:
Done, including using a constant for the sequence name.